### ISSUE TITLE
Unix timestamp output?

### ISSUE THREAD

**Issue by quantumleap33** [amsul/pickadate.js#842]
Is it possible to output a unix timestapm integer instead of a date string?

**Comment by jasontrask** [comment]
I don't think this is documented, but if use the `get` method with an _item_ 'thing' (the * ones in the [docs](http://amsul.ca/pickadate.js/api/#method-get)), but not providing a format, you'll get back the unix timestamp expressed in milliseconds. Just divide that by 1000 to get your unix timestamp.
e.g.
`picker.get('select')/1000`
